src/css/parse.js

```diff
@@ -1203,75 +1249,19 @@ function checkFunction(i) {
  * @return {Node}
  */
 function getFunction() {
-  let type = NodeType.FunctionType;
-  let token = tokens[pos];
-  let line = token.ln;
-  let column = token.col;
-  let ident = getIdent();
-  let content = [ident];
+  const type = NodeType.FunctionType;
+  const token = tokens[pos];
+  const line = token.ln;
+  const column = token.col;
+  let content = [getIdent()];
+  let body = getArguments();
 
-  content.push(getArguments());
+  content.push(body);
```

Comment:
Haha looking at it now yeah it's a bit weird - some of these changes are just to remove the difference from the `scss` parser
 ```js
...
let body;

  body = getArguments();

  x.push(body);
...
```

I do think your solution is cleaner though so I'll implement that instead :+1: